src/netex-convertor/handler.test.ts

```diff
@@ -97,6 +98,22 @@ describe('netexConvertorHandler', () => {
 
     it('should generate single fare netex with no undefined variables', async () => {
         netexGeneratorSpy.mockRestore();
+        dbSpy.mockImplementation(() =>
+            Promise.resolve([
+                {
+                    nocCode: 'MCTR',
```

Comment:
Why are these alternative operator details required (rather than those specified in the `beforeEach` block? Is there something specific being done with the nocCode?

Reply:
Actually ignore that - looks like the NOC code needs to match the test data ticket of that type - all good!